docs/packaging/versioning.md

```diff
@@ -212,7 +212,29 @@ A few ways to look up the version of an installed package are:
 
 ## Native Linux package versions
 
-TODO: fill this in together with https://github.com/ROCm/TheRock/pull/2159
+TheRock supports rpm and debian packages. Each has different versioning scheme as mentioned below.
+Native package versions are handled by scripts:
+
+- [`build_tools/compute_rocm_native_package_version.py`](/build_tools/compute_rocm_native_package_version.py)
+  - [`build_tools/tests/compute_rocm_native_package_version_test.py`](/build_tools/tests/compute_rocm_native_package_version_test.py)
+
+The script produces these versions for rpm packages for each distribution channel:
+
+| Distribution channel | Version format              | Version example                                                                                                                        |
+| -------------------- | --------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| stable releases      | `X.Y.Z`                     | `7.10.0`                                                                                                                               |
+| prereleases          | `X.Y.Z~rcN`                 | `7.10.0i~rc0`<br>(The first release candidate for that stable release)                                                                 |
```

Comment:
Should the `i` be in this `7.10.0i` version example, or is that a typo? I don't see `i`. in the version format or in the script that generates the versions.
```suggestion
| prereleases          | `X.Y.Z~rcN`                 | `7.10.0~rc0`<br>(The first release candidate for that stable release)                                                                 |
```